README.md

```diff
@@ -54,7 +58,7 @@ Running with no command-line options at all is like running `nca.py --sanity k8s
 - `--daemon`\
   Run NCA as a daemon. Send and receive data using a REST API.
 - `--output_format <format>`\
-  Output format specification (txt/yaml/csv).\
+  Output format specification (txt/yaml/csv/md).\
   *default:* txt\
   *shorthand:* `-o`
 - `--file_out <file name>`\
```

Comment:
Can you please also document the `--pr_url` flag I just added?
Its argument should be a URL pointing to the relevant `comments` resource in the GitHub API.
e.g., `https://api.github.com/repos/shift-left-netconfig/online-boutique/issues/1/comments`

Reply:
done